lib/lte_link_control/lte_lc.c

```diff
@@ -47,9 +47,10 @@ LOG_MODULE_REGISTER(lte_lc, CONFIG_LTE_LINK_CONTROL_LOG_LEVEL);
 #define AT_XSYSTEMMODE_RESPONSE_PREFIX		"%XSYSTEMMODE"
 #define AT_XSYSTEMMODE_PROTO			"AT%%XSYSTEMMODE=%d,%d,%d,%d"
 /* The indices are for the set command. Add 1 for the read command indices. */
-#define AT_XSYSTEMMODE_LTEM_INDEX		0
-#define AT_XSYSTEMMODE_NBIOT_INDEX		1
-#define AT_XSYSTEMMODE_GPS_INDEX		2
+#define AT_XSYSTEMMODE_READ_LTEM_INDEX		1
```

Comment:
supernit: ```LTEM``` could be written as ```LTE_M``` to align with the already used naming convention in this lib.

Reply:
It seems that lte_lc.c/h uses `LTEM `and Kconfig uses `LTE_M`. I guess we should follow the same rule with the new implementation, unless we are going to align everything either way.

Reply:
Yeah, it seems that the different versions of `LTEM` and `LTE_M` are consistently used in either lte_lc.c/.h and Kconfig. I'll leave it as is in this PR, but might fix in follow-up PR